teams/teams-ps/teams/Grant-CsGroupPolicyPackageAssignment.md

```diff
@@ -0,0 +1,109 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Grant-CsGroupPolicyPackageAssignment
+author: icchan
+ms.author: icchan
+manager: dasosby
+online version: https://docs.microsoft.com/powershell/module/teams/grant-csgrouppolicypackageassignment
+schema: 2.0.0
+---
+
+# Grant-CsGroupPolicyPackageAssignment
+
+## SYNOPSIS
+
+**Note:** This cmdlet is currently only available in private preview in the pre-release Teams PowerShell module.
+
+This cmdlet assigns a policy package to a group in a tenant.
+
+## SYNTAX
+
+```
+Grant-CsGroupPolicyPackageAssignment -GroupId <String> -PackageName <String> [-PolicyRankings <String[]>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet assigns a policy package to a group in a tenant. The available policy packages and their definitions can be found by running Get-CsPolicyPackage. Policy rankings can be optionally specified to determine which policies in a package will be assigned to the user in case they belong to two or more groups. If policy rankings for a policy type is not specified, the group's policy assignment for that policy type will be ranked last. Finally, if a user was directly assigned a package, direct assignment takes precedence over group assignment. For more information on policy rankings and group policy assignments, please review [the description section under New-CsGroupPolicyAssignment](New-CsGroupPolicyAssignment.md#description). For more information on policy packages, please review https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages.
```

Comment:
```suggestion
This cmdlet assigns a policy package to a group in a tenant. The available policy packages and their definitions can be found by running Get-CsPolicyPackage. Policy rankings can be optionally specified to determine which policies in a package will be assigned to the user in case they belong to two or more groups. If policy rankings for a policy type is not specified, the group's policy assignment for that policy type will be ranked last. Finally, if a user was directly assigned a package, direct assignment takes precedence over group assignment. For more information on policy rankings and group policy assignments, please review [the description section under New-CsGroupPolicyAssignment](New-CsGroupPolicyAssignment.md#description). For more information on policy packages, please review [Manage policy packages in Microsoft Teams](https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages).
```